scripts/docker_build_wheel.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+# Copyright 2020 The ElasticDL Authors. All rights reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+export BASE_IMAGE=tensorflow/tensorflow:2.1.0-py3
+
+docker build --target dev -t elasticdl:dev -f elasticdl/docker/Dockerfile \
+    --build-arg BASE_IMAGE="$BASE_IMAGE" .
+
+docker run --rm -it --net=host -v "$PWD":/work -w /work elasticdl:dev \
+    bash -c "scripts/build_and_test.sh 0"
```

Comment:
It is not common to use 1 and 0 as command-line options to a script. It seems that we should have

- build.sh, and 
- test.sh

where test.sh calls build.sh before testing.